bin/thor

```diff
@@ -47,6 +62,21 @@ var cluster = require('cluster')
   , received = 0
   , robin = [];
 
+//
+// Process headers
+//
+var headers = {};
+cli.header.forEach(function(header){
+  var keyValue = header.split(':');
```

Comment:
Header Val might have ':' therefore the split will be wrong. Try:
var keyValue = header.split(/:/);
var key = keyValue.shift();
var value = keyValue.join(':');